lib/events/events.proto

```diff
@@ -1143,6 +1143,7 @@ message OneOf {
         events.AppSessionStart AppSessionStart = 37;
         events.AppSessionChunk AppSessionChunk = 38;
         events.AppSessionRequest AppSessionRequest = 39;
+        events.SessionUpload SessionUpload = 40;
```

Comment:
use the same field IDs as on `master`, to allow 5.x binaries to communicate with 6.x
```suggestion
        events.SessionUpload SessionUpload = 43;
```